piqe_ocp_lib/api/resources/ocp_routes.py

```diff
@@ -10,20 +10,11 @@ class OcpRoutes(OcpBase):
     """
     OcpPods Class extends OcpBase and encapsulates all methods
     related to managing Openshift pods.
-    :param hostname: (optional | str) The hostname/FQDN/IP of the master
-                     node of the targeted OCP cluster. Defaults to
-                     localhost if unspecified.
-    :param username: (optional | str) login username. Defaults to admin
-                      if unspecified.
-    :param password: (optional | str) login password. Defaults to redhat
-                      if unspecified.
-    :param kube_config_file: A kubernetes config file. It overrides
-                             the hostname/username/password params
-                             if specified.
+    :param kube_config_file: A kubernetes config file.
     :return: None
     """
-    def __init__(self, hostname='localhost', username='admin', password='redhat', kube_config_file=None):
-        super(OcpRoutes, self).__init__(hostname, username, password, kube_config_file)
+    def __init__(self, kube_config_file=None):
+        super(OcpRoutes, self).__init__(kube_config_file=kube_config_file)
```

Comment:
Will it not break the existing libs/tests?. Because existing libs/tests may be passing hostname, username, password

Reply:
It won't break since ocp_base has host/user/password has default values. Infact when i will submit another PR for dyn_client singleton, We are planning to remove those fields.